pkg/client/client.go

```diff
@@ -145,33 +149,33 @@ func removeSlash(path string) string {
 	return strings.TrimRight(path, "/")
 }
 
-func (c *client) get(rawURL string) ([]byte, int, error) {
+func (c *client) get(rawURL string) ([]byte, int, Error) {
 	u, err := url.Parse(rawURL)
 	if err != nil {
-		return nil, 0, err
+		return nil, 0, NewError("ErrHTTPInternal", "Something goes wrong", err)
```

Comment:
Instead of `Something goes wrong` can we provide a more helpful message like `HTTP GET to %s failed"?

Reply:
is error code ok `ErrHTTPInternal` ?